core_tic_tac_toe/display.py

```diff
@@ -7,27 +7,24 @@ def print_board(self, moves):
         horizontal_bar = self.__horizontal_bar()
         vertical_bars = self.__vertical_bars()
 
-        board = (
-            vertical_bars +
-            "  {}  |  {}  |  {} \n".format(moves[0], moves[1], moves[2]) +
-            vertical_bars +
-            horizontal_bar +
-            vertical_bars +
-            "  {}  |  {}  |  {} \n".format(moves[3], moves[4], moves[5]) +
-            vertical_bars +
-            horizontal_bar +
-            vertical_bars +
-            "  {}  |  {}  |  {} \n".format(moves[6], moves[7], moves[8]) +
-            vertical_bars
-        )
+        board = ""
+
+        for idx, move in enumerate(moves):
+            board += vertical_bars
+            board += self.__generate_rows(move)
+            board += vertical_bars
+
+            if idx != len(moves) - 1:
+                board += horizontal_bar
 
         self.io.print_message(board)
 
     def print_invalid_move_message(self):
         self.io.print_message("Invalid position, please try again!")
 
     def get_player_input(self):
-        return self.io.get_player_input()
+        user_move = self.io.get_player_input()
+        return user_move
```

Comment:
Why use `user_move` here?

Reply:
I had a bug in one of my tests regarding mockIo, so I wanted to print the result of calling `get_player_input`. Which is why I created the `user_move` variable so I could store the result and print it.